server/config.js

```diff
@@ -2,7 +2,13 @@
 
 const path = require('path');
 const log = require('./logger');
-let config = require(path.join(process.cwd(), 'stubborn'));
+let config;
+
+try {
+  config = require(path.join(process.cwd(), 'stubborn'));
+} catch (e) {
+  config = {};
```

Comment:
`log.error(\`cannot find "stubborn.js" or "stubborn.json" in ${process.cwd()}\`)` might be a good warning to put here